doc/filters/format_datetime.rst

```diff
@@ -40,6 +40,30 @@ By default, the filter uses the current locale. You can pass it explicitly:
     {# 7 août 2019 23:39:12 #}
     {{ '2019-08-07 23:39:12'|format_datetime(locale='fr') }}
 
+It is possible to set TimeZone (see `WiKi: List of tz database time zones
+<https://en.wikipedia.org/wiki/List_of_tz_database_time_zones>`_
+). Date and time for timezone set via argument calculated from default timezone, default timezone depends from `system settings
+<https://twig.symfony.com/doc/1.x/filters/date.html#timezone>`_
```

Comment:
this should be using a `:ref:` link so that it looks to the same version of the Twig doc